assets/css/_notification_drawer.scss

```diff
@@ -13,6 +13,13 @@
   @media screen and (max-width: $mobile-max-width) {
     width: 100%;
   }
+  @media screen and (max-width: $mobile-max-width) {
```

Comment:
Is `$mobile-max-width` actually the correct value? It's defined in `app.scss` as 768px and it was a breakpoint used in most old Skate designs, but it's not the same as the breakpoint for mobile portrait nav.

On the other hand, if it is supposed to be `$mobile-max-width`, this can be combined with the above selector / block.

Reply:
Good call this is not the right value - as-is, shrinking to a window between 480-768px wide produces a broken mashup of navs. I'll introduce a new `$mobile-portrait-max-width` variable and reference that instead.

Reply:
For what it's worth, the breakpoints for the new nav are defined in `src/hooks/useDeviceType.ts`. Since the nav logic happens largely in JavaScript, I didn't end up creating new SCSS constants for the breakpoints. I think eventually we want to retire the use of the existing `$mobile-max-width` and make all of the different breakpoints consistent.